project/src/main/scala/org/broadinstitute/clio/publish/ArtifactoryFatJarPublishingPlugin.scala

```diff
@@ -30,9 +30,11 @@ object ArtifactoryFatJarPublishingPlugin extends AutoPlugin {
   private val ivyXmlPattern =
     "[organization]/[module]/[revision]/[type]s/ivy-[revision].xml"
   private val ivyArtifactoryPatterns = Patterns(
-    ivyPatterns = Seq(ivyXmlPattern),
-    artifactPatterns = Seq(ivyArtifactPattern),
-    isMavenCompatible = true
+    ivyPatterns = Vector(ivyXmlPattern),
+    artifactPatterns = Vector(ivyArtifactPattern),
+    isMavenCompatible = true,
+    descriptorOptional = true,
+    skipConsistencyCheck = false
```

Comment:
Were there equivalent options for `descriptorOptional` and `skipConsistencyCheck` in 0.13 that these are now matching? Or are these brand new?